src/lib/layouts/Section.svelte

```diff
@@ -1,6 +1,6 @@
 <section
 	id={$$props.id}
-	class="py-16 child:mx-16 md:child:mx-32 xxl:child:!mx-auto xxl:child:max-w-screen-2xl
+	class="py-16 child:mx-8 md:child:mx-32 xxl:child:!mx-auto xxl:child:max-w-screen-2xl
```

Comment:
I'm not really for that change, and I didn't see why it's even needed

If you want to overwrite this for one section, prefer doing `<Section class="child:!mx-8">`

Reply:
After testing i chose your solution